### PR TITLE
botvs: Fix lint

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: types/botvs/index.d.ts [hunk 1]
```diff
@@ -392,59 +377,43 @@ declare global {
              * 支持现货(火币/BitVC/OKCoin/OKCoin国际/OKCoin期货/BTCChina/BitYes)市价单, 市价单价格指定为-1
              * exchange.Buy(1000), 指买市价1000元的币, BTCChina例外exchange.Buy(0.3)指市价买0.3个币
              *
-             * @param {number} Price
-             * @param {number} Amount
-             * @param {*[]} args
-             * @return {string} - 订单ID
+             * @return - 订单ID
```
Comment (atd-schubert) [inline types/botvs/index.d.ts]: I have refactor this some minutes before in #20932, but I have not take care about the comments! Should I merge this PR into mine and solve the conflicts?

**Review by atd-schubert — COMMENTED** [review]
I had update this type definition minutes ago. Should I merge the changes?

File: types/botvs/index.d.ts (hunk 1 above)
Comment (andy-ms) [reply types/botvs/index.d.ts]: :+1:

**Comment by acrazing** [comment]
Why do you remove the JSDoc comments?

**Comment by atd-schubert** [comment]
Let's take a look at this example from the first lines:
```typescript
/**
 * 返回交易所名称(string)
 *
 * @return {string}		
 */
GetName(): string;
```

`GetName` returns a string according to typescript. In addition there is a JSDoc comment that specifies that it will return a string. We don't need this information, because it is redundant.

The other comments wasn't removed, only the redundant type information... There is also a new linting-rule in tslint (https://github.com/palantir/tslint/pull/2754) for that. This is why it causes trouble, because when you change a type-definition, maybe the typescript-bot will fail (within another definition). The rule is quite new to tslint. In the past it was allowed, but it was also not a good practice...

Tslint getting more and more strict.

**Comment by acrazing** [comment]
TypeScript's types definition is not enough to hint types, especially for the function's optional parameters, it could not indicate the default values, for example:

with jsdoc:

with-jsdoc

without jsdoc:

no-jsdoc

And more, almost all of the code is generated by some script~

**Comment by atd-schubert** [comment]
The information is still redundant. In addition, we do not remove comments, just types!

Take a look at [this line](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/20933/files/c319302ff23dab1fa22e68ce8c2953c6d873bac2#diff-c0636e381292513ff8243ece8949d157R2827):

There is still the information about the default value, but no information about the type in JSDoc.
You can also put default values to typescript in the es6 way. In a short summary: You use TypeScript to add data-types. When you use a second time data-types in JSDoc you make the type-definition twice and this is redundant!

**Pull request closed without merging**